tensorflow/compiler/mlir/tools/kernel_gen/tf_to_kernel.cc

```diff
@@ -189,6 +184,10 @@ int main(int argc, char** argv) {
       "unroll_factors",
       llvm::cl::desc("factors to unroll by, separated by commas"),
       llvm::cl::ZeroOrMore, llvm::cl::CommaSeparated);
+  llvm::cl::opt<bool> jit_compile(
```

Comment:
`jit_compile` -> `jit_i64_indexed_for_large_tensors`

Does this compile?

Reply:
fixed.